tools/topology/sof-tgl-max98373-rt5682.m4

```diff
@@ -20,17 +20,34 @@ include(`platform/intel/dmic.m4')
 
 DEBUG_START
 
+define(CHANNELS, `4')
+
+define(KFBM_TYPE, `vol-kfbm')
+
+define(DMIC_PCM_48k_ID, `99')
+define(DMIC_PIPELINE_48k_ID, `4')
+define(DMIC_DAI_LINK_48k_ID, `1')
+
+define(DMIC_PCM_16k_ID, `100')
+define(DMIC_PIPELINE_16k_ID, `9')
+define(DMIC_PIPELINE_KWD_ID, `10')
+define(DMIC_DAI_LINK_16k_ID, `2')
+
+define(KWD_PIPE_SCH_DEADLINE_US, 20000)
+
+include(`platform/intel/intel-generic-dmic-kwd.m4')
+
 #
 # Define the pipelines
 #
 # PCM0 ---> volume ----> SSP1  (Speaker - max98373)
 # PCM1 <---> volume <----> SSP0  (Headset - ALC5682)
-# PCM99 <---- volume <---- DMIC01 (dmic0 capture)
 # PCM2 ----> volume -----> iDisp1
 # PCM3 ----> volume -----> iDisp2
 # PCM4 ----> volume -----> iDisp3
 # PCM5 ----> volume -----> iDisp4
-
+# PCM99 <---- volume <---- DMIC01 (dmic 48k capture)
+# PCM100 <---- kpb <---- DMIC16K (dmic 16k capture)
```

Comment:
What's the rational for using this large gap again ? Is this to avoid collisions with pipelines we include ?

Reply:
This is request from @RDharageswari  No idea why these numbers are chosen.
@RDharageswari  can you explain here?

Reply:
Yes, just to avoid collision with other pipelines.. Also newly added DMIC16K is adding next to it.